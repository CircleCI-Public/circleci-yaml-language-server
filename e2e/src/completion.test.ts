import { Commands } from './types';
import {
  configFileUri,
  command,
} from './helpers';

describe('Completion command', () => {
  it('Completion command', async () => {
    const response = await command(
      Commands.Completion,
      {
        position: {
          character: 1,
          line: 1,
        },
        textDocument: {
          uri: configFileUri('config1.yml'),
        },
      },
    );

    expect(response).toMatchSnapshot();
  });
});
