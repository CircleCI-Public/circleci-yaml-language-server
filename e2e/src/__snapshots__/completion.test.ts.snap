// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Completion command Completion command 1`] = `
{
  "isIncomplete": true,
  "items": [],
}
`;
